Reportedly/Application/Constants/TypeAliases.swift

```diff
@@ -16,6 +16,8 @@ typealias ResultRestfulCompletion   = (Result<ResponseSuccess, ResponseError>) -
 typealias ResponseCompletion        = (_ data: Data?, _ success: ResponseSuccess?, _ error: ResponseError?) -> Void
 typealias StringCompletion          = (String) -> Void
 typealias SuccessCompletion         = (Bool) -> Void
+typealias ResultUserCompletion      = (Result<User, ResponseError>) -> Void
+typealias ResultReportsCompletion   = (Result<[Report], ResponseError>) -> Void
```

Comment:
I think we could perhaps use generic `typealias ResultCompletion<T> = (Result<T,>) -> Void` instead of ResultUserCompletion and ResultReportsCompletion? What do you think? 🤔

Reply:
nice suggestion, let me update 👍